static/app/components/events/eventReplay/replayPreview.tsx

```diff
@@ -45,9 +48,24 @@ function ReplayContent({orgSlug, replaySlug, event}: Props) {
   }, [eventTimestamp, startTimestampMs]);
 
   if (fetchError) {
+    // TODO(replay): this is kind of a lot of writing. When the product docs launch in GA we will link to them instead.
+    const reasons = [
+      t('The Replay is still processing and is on its way'),
+      t('The Replay has been deleted'),
+      t('There is an internal systems error or active issue'),
+    ];
+
     return (
       <Alert type="info" showIcon data-test-id="replay-error">
-        {t('The replay associated with this event could not be found.')}
+        <p>
+          {t('The replay associated with this event could not be found.')}{' '}
```

Comment:
Casing of replay here is inconsistent

Reply:
This case looks correct imo: https://www.notion.so/Session-Replay-Lexicon-Style-Guide-bcbbd569e5fc41f2aaff2f7d796fd401

`replay` is the record/event name, and it's talking about db rows associating.

Reply:
the bullet points are all wrong though

Reply:
Ah yeah was trying to reference these as well: https://github.com/getsentry/sentry/pull/44000/files#diff-2cd79b25cb39dfbf355e15e0c580a785d9df6a0f0bbd4c1269e0209e25373c42R53-R54